dags/iop/parser.py

```
@@ -75,3 +80,13 @@ def _get_related_article_doi(self, article):
         except AttributeError:
             self.logger.error("No related article dois found", dois=self.dois)
             return
+
+    def _get_arxiv_eprints(self, article):
+        arxivs_value = article.find(
+            "front/article-meta/custom-meta-group/custom-meta/meta-value"
+        )
+        try:
+            pattern = re.compile(r"(arxiv:|v[0-9]$)", flags=re.I)
+            return [{"value": pattern.sub("", arxivs_value.text.lower())}]
```

I would prefer to create a variable with proper formatting of an eprint  (so this `pattern.sub("", arxivs_value.text.lower())`) and then return, it will be more readable

---

Also, you are not checking if the extracted value matches the pattern, if you l run `re.sub` on whatever value, it will always be returned, which is incorrect, unless you don't care at this point about an incorrect arxiv and you want to raise an error later during validation